bundles/org.openhab.binding.miio/src/main/resources/database/ijai.vacuum.v19-miot.json

```diff
@@ -0,0 +1,2543 @@
+						"parameterType":"STRING"
+					}
+				]
+			},
+			{
+				"property":"restrict-points",
+				"siid":9,
+				"piid":3,
+				"friendlyName":"Point Zone - Restrict Points",
+				"channel":"restrict_points",
+				"type":"String",
+				"refresh":false,
+				"actions":[
+					{
+						"command":"set_properties",
+						"parameterType":"STRING"
+					}
+				]
+			},
+			{
+				"property":"target-point",
+				"siid":9,
+				"piid":5,
+				"friendlyName":"Point Zone - Target Point",
+				"channel":"target_point",
+				"type":"String",
+				"refresh":false,
+				"actions":[
+					{
+						"command":"set_properties",
+						"parameterType":"STRING"
+					}
+				]
+			},
+			{
+				"property":"remember-state",
+				"siid":10,
+				"piid":1,
+				"friendlyName":"Map - Remember State",
+				"channel":"remember_state",
+				"type":"Switch",
+				"refresh":false,
+				"actions":[
+					{
+						"command":"set_properties",
+						"parameterType":"ONOFFNUMBER"
+					}
+				]
+			},
+			{
+				"property":"cur-map-id",
+				"siid":10,
+				"piid":2,
+				"friendlyName":"Map - Cur Map Id",
+				"channel":"cur_map_id",
+				"type":"Number",
+				"stateDescription":{
+					"minimum":0,
+					"step":1,
+					"pattern":"%.0f",
+					"readOnly":true
+				},
+				"refresh":false,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"map-num",
+				"siid":10,
+				"piid":3,
+				"friendlyName":"Map - Map Num",
+				"channel":"map_num",
+				"type":"Number",
+				"stateDescription":{
+					"minimum":0,
+					"maximum":5,
+					"step":1,
+					"pattern":"%.0f",
+					"readOnly":true
+				},
+				"refresh":false,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"cur-cleaning-path",
+				"siid":10,
+				"piid":5,
+				"friendlyName":"Map - Cur Cleaning Path",
+				"channel":"cur_cleaning_path",
+				"type":"String",
+				"stateDescription":{
+					"readOnly":true
+				},
+				"refresh":false,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"build-map",
+				"siid":10,
+				"piid":14,
+				"friendlyName":"Map - Build Map",
+				"channel":"build_map",
+				"type":"Number",
+				"stateDescription":{
+					"readOnly":true,
+					"options":[
+						{
+							"value":"0",
+							"label":"None"
+						},
+						{
+							"value":"1",
+							"label":"Build"
+						},
+						{
+							"value":"2",
+							"label":"Clean"
+						}
+					]
+				},
+				"refresh":false,
+				"actions":[
+					
+				],
+				"readmeComment":"Value mapping [\"0\"\u003d\"None\",\"1\"\u003d\"Build\",\"2\"\u003d\"Clean\"]"
+			},
+			{
+				"property":"has-new-map",
+				"siid":10,
+				"piid":19,
+				"friendlyName":"Map - Has New Map",
+				"channel":"has_new_map",
+				"type":"Contact",
+				"stateDescription":{
+					"readOnly":true
+				},
+				"refresh":false,
+				"actions":[
+					
+				]
+			},
+			{
+				"property":"dnd-enable",
+				"siid":12,
+				"piid":1,
+				"friendlyName":"Disturb - Dnd Enable",
+				"channel":"dnd_enable",
+				"type":"Contact",

```

Comment:
should this not be a switch? Is the contact working properly?

Reply:
From what I saw in openhab's documentation, a `Contact` is basically a read-only `Switch`. It seems to work properly, but it does not show a toggle switch, just a text with OPENED/CLOSED. But when I analyze the item, I see the on/off pattern fine.
![image](https://user-images.githubusercontent.com/831119/168387583-741e647b-3233-47f1-8e95-1e15435edd69.png)

![image](https://user-images.githubusercontent.com/831119/168387512-1b453672-07bf-4dcc-9a19-6cef7ea25dfc.png)